### ISSUE TITLE
how to get pdb_labels training dataset

### ISSUE THREAD

**Issue by ssn1771** [dptech-corp/Uni-Fold#55]
halo, i can get the pdb_features dataset from the script in Uni-Fold-jax program, but i can not find any code about gererating the label data, I’d appreciate some help.

**Comment by ZiyaoLi** [comment]
Currently we do not release the scripts, because it has heavy dependency on how you organize the label data. We are soon releasing the entire PDB training set, perhaps in 1-2 weeks so maybe you can wait and use it.

**Comment by ZiyaoLi** [comment]
see the updates of datasets in #28.
